pkg/network/networknamescheme/networknamescheme.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package networknamescheme
```

Comment:
We are already under the `network` package, so just using `namescheme` may fit better.
I do see a problem for the long run with this new package... it may become overloaded with all kind of unrelated names.

But I do not know if I have a better place for this naming constants.
The closes I can think of is under `network.link`, so it will be `network.link.namescheme`. WDYT?
I must admit I am not really sure my suggestion fits.

Reply:
I agree, but I don't think your suggestion makes sense I'm afraid..

Changed to `namescheme`